scripts/coco

```diff
@@ -1,40 +1,151 @@
 #!/usr/bin/env python3.7
 
 import argparse
+import json  # orjson does not have an option to pretty-serialize (indent=2)
 import os
 import sys
+import yaml
 
-from coco import Master
+from coco import Endpoint, result, config
 
-def start(args):
-    master = Master(args.conf)
+# result printing styles:
+STYLES = ["json", "yaml"]
 
-# The default coco.conf path.
-# The default here should be suitable for running coco
-# without installing it. An installed coco should always
-# specify the location of its configuration file
-# on the command line.
-coco_conf = os.path.normpath(os.path.join(sys.path[0], "../conf/coco.conf"))
 
-parser = argparse.ArgumentParser(
-        description="This is coco (Config Control).",
-        epilog="""\
-If no configuration file is specified via -c, coco will use {0} which should be suitable for
-running coco without first installing it.
-""".format(coco_conf)
-        )
+def str2bool(v):
+    if v.lower() in ('yes', 'true', 't', 'y', '1'):
+        return True
+    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+
+
+def read_config(config_path):
```

Comment:
This entire function is unused, and I keep forgetting to remove it. Might as well do it now.